sources/editor/Stride.Core.Assets.Editor/ViewModel/AssetCollectionViewModel.cs

```diff
@@ -244,9 +244,6 @@ public AssetCollectionViewModel([NotNull] IViewModelServiceProvider serviceProvi
 
         public int AssetCount => assets.Count;
 
-        // FIXME: this property was added because for some reason DataGridEx has a lot of issues when displaying both assets and folders
```

Comment:
I'm a little worried about this comment here, but my understanding is that this method was no longer referenced for you to remove it, so maybe the comment is obsolete.

Reply:
what should i do?

Reply:
it was unused and DataGridEx is empty.. for "future developement".

Reply:
Isn't that property still used over here ? https://github.com/stride3d/stride/blob/a19a8fb9a2af3e79b61c4d3185efcf5b3e3cbd6a/sources/editor/Stride.Core.Assets.Editor/View/AssetViewUserControl.xaml#L181-L187
Or is that stuff also obsolete ?